### PR TITLE
Completed prep-css-flexbox exercise

### PR DESCRIPTION
**Pull request by marissalortega**
Screen Shot 2021-09-16 at 9 51 30 PM
Screen Shot 2021-09-17 at 10 26 55 AM
Screen Shot 2021-09-17 at 10 33 11 AM
Screen Shot 2021-09-17 at 10 38 33 AM

### CODE REVIEW

File: prep-css-flexbox/display-flex.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+    }
+
+    #card-deck {
+      padding: 20px;
+    }
```
Comment (scbowler) [inline prep-css-flexbox/display-flex.html]: This should be combined into a single CSS ruleset:

```CSS
#card-deck {
  display: flex;
  padding: 20px;
}
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one small update needed. Let me know if you have any questions.

File: prep-css-flexbox/display-flex.html (hunk 1 above)
Comment (marissalortega) [reply prep-css-flexbox/display-flex.html]: Screen Shot 2021-09-17 at 4 34 43 PM

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**